pkg/quay/quay_test.go

```diff
@@ -262,3 +263,132 @@ func TestQuayClient_handleRobotName(t *testing.T) {
 		})
 	}
 }
+
+func TestQuayClient_GetTagsFromPage(t *testing.T) {
+	defer gock.Off()
+
+	org := "test_org"
+	repo := "test_repo"
+
+	testCases := []struct {
+		name          string
+		pages         int
+		tagsPerPage   int
+		totalTags     int
```

Comment:
Do we require `totalTags` to be here?

Reply:
You're right, it's no longer necessary. Thank you